test/mlir/accelerators/nnpa/transform/zhigh-constant-propagation-be/constprop.mlir

```diff
@@ -9,7 +9,7 @@ func.func @remove_stick_1d() -> tensor<6xf32, #zhigh.encoding<{dataLayout = "1D"
+  // CHECK-NEXT: %0 = "zhigh.StickifiedConstant"() {alignment = 4096 : i64, value = dense_resource<zhigh> : tensor<4096xi8>} : () -> tensor<6xf32, #zhigh.encoding<{dataLayout = "1D"}>>
```

Comment:
Is it possible to check the value of `dense_resource<zhigh>`? We want to check if the generated stickified constant is correct or not in this test. I guess we can use the old value with alignment. But, if it does not work, I can help generate values on a BE machine.

Reply:
I updated the tests and I think they should work now.